### PR TITLE
Mfoggia/add column alignment custom rule

### PR DESCRIPTION
**Pull request by MrMarCode**

### CODE REVIEW

File: .markdownlint-cli2.shared.cjs [hunk 1]
```diff
@@ -1,6 +1,9 @@
 'use strict';
 
+const indentAlignment = require('./node_modules/@silvermine/markdownlint-rule-indent-alignment/src/indent-alignment.js')
```
Comment (onebytegone) [inline .markdownlint-cli2.shared.cjs:3]: Does this work?

```suggestion
const indentAlignment = require('@silvermine/markdownlint-rule-indent-alignment')
```

**Review by onebytegone — CHANGES_REQUESTED** [review]

File: .markdownlint.json [hunk 2]
```diff
@@ -6,6 +6,11 @@
    "ul-style": {
       "style": "asterisk"
    },
+
+   "indent-alignment": {
+      "ul_indent": 3
+   },
+
```
[the remark this answers is not in the document]
Comment (MrMarCode) [reply .markdownlint.json:13]: Yes, from my testing it seems that unused rule configs are ok.

File: .markdownlint-cli2.shared.cjs [hunk 3]
```diff
@@ -1,7 +1,11 @@
 'use strict';
 
+const indentAlignment = require('@silvermine/markdownlint-rule-indent-alignment')
```
Comment (onebytegone) [inline .markdownlint-cli2.shared.cjs:3]:
```suggestion
const indentAlignment = require('@silvermine/markdownlint-rule-indent-alignment');
```

File: .markdownlint.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (onebytegone) [reply .markdownlint.json:13]: The requested change for the extra whitespace?

**Review by onebytegone — CHANGES_REQUESTED** [review]

**Review by onebytegone — APPROVED** [review]

**Pull request merged**